### ISSUE TITLE
output of dj.kill has no particular order

### ISSUE THREAD

**Issue by ecobost** [datajoint/datajoint-python#668]
For instance, 
```python3
In [22]: dj.kill()
  ID USER         STATE         TIME  INFO
+--+ +----------+ +-----------+ +--+
1241912 ewang                      2261  None
1266899 root                      32096  None
1241936 ewang                      7988  None
1267543 jiakun                     6088  None
1267496 zhiwei                     7736  None
1238807 atlab                         0  None
1238804 atlab                         0  None
1238800 atlab                         0  None
1267558 jiakun                      174  None
1265835 eywalker                  70459  None
1265570 manolis                   54718  None
1267661 ewang                      1434  None
...
```
It would be nice if it was ordered either by id or by user.

**Comment by ixcat** [comment]
normally `dj.kill()` only can see the current user so this is less of an issue - for this installation, process privilege was added to allow users to see all queries. The actual query is in `datajoint/admin.py` and is currently: `select * from information_schema.processlist where id <> connection_id()`. Not sure what or how mysql chooses to sort this internally. Agree this could be sorted. Not sure what would be best. Any feedback on that e.g. what sorts of scenarios is this useful?

To note: the function will take an optional string-style restriction, so e.g. `dj.kill('user like "ecobost"')` or `dj.kill('time > 3600')` may be useful to narrow things down.

**Comment by ecobost** [comment]
`ORDER BY connection_id` should be ok, don't think there's a right answer

**Comment by dimitri-yatsenko** [comment]
Let's just add an `order_By` keyword argument similar to the one in fetch. I see how this can be useful

**Comment by eywalker** [comment]
Fixed by #775
